Reflections/Week_5_reflection.md

```diff
@@ -0,0 +1,28 @@
+## Accomplishments
+
+I have successfully added in basic file I/O to my program.
+
+It is imperfect, I am still not serializing all of the features of my lines as-is and more work will need to be done to support that, however I am happy with being able to save and load at all.
```

Comment:
This is fantastic! Coping with binary formats is always rough; I'd like to share the MagicaVoxel file format for inspiration: [here](https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt). Chunk formats are pretty common when dealing with binary data!